libnavui-androidauto/build.gradle

```diff
@@ -41,13 +41,10 @@ dependencies {
     // This defines the minimum version of Maps, Navigation, and Search which are included in
     // this SDK. To upgrade the SDK versions, you can specify a newer version in your downstream
     // build.gradle.
-    def carNavVersion = "2.8.0-rc.1"
+    def carNavVersion = "2.9.0-alpha.3"
     def carSearchVersion = "1.0.0-beta.36"
     api("com.mapbox.search:mapbox-search-android:${carSearchVersion}")
-
-    // TODO use the carNavVersion when 2.9.0-alpha.2 is available
-    api(project(":libnavigation-android"))
-//    api("com.mapbox.navigation:android:${carNavVersion}")
+    api("com.mapbox.navigation:android:${carNavVersion}")
```

Comment:
We used to have a project dependency with debug build type (#6316), have we abandoned this approach?

Reply:
We needed that in order to support incompatible releases. When we push code that is incompatible it also blocks releases. This is a big part of the reason to ensure compatible apis, so that we can always have a release pipeline that is not blocked.

Reply:
Need to remove the `api` from the search dependency though